Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -117,3 +119,33 @@ public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenSt
         errorEvents?.fire(error.networkProtectionError)
     }
 }
+
+#else
+
+public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenStore {
+    private let accessTokenProvider: () -> String?
+
+    public static var authTokenPrefix: String { "ddg:" }
+
+    public init(accessTokenProvider: @escaping () -> String?) {
+        self.accessTokenProvider = accessTokenProvider
+    }
+
+    public func store(_ token: String) throws {
+        assertionFailure("Unsupported operation")
+    }
+
+    public func fetchToken() throws -> String? {
+        accessTokenProvider().map { makeToken(from: $0) }
+    }
+
+    public func deleteToken() throws {
+        assertionFailure("Unsupported operation")
+    }
+
+    private func makeToken(from subscriptionAccessToken: String) -> String {
+        Self.authTokenPrefix + subscriptionAccessToken
+    }
+}
```

Comment:
Ideally we can remove this one day since it's just a fancy wrapper around the access token store, but it's fine to keep for now.